### PR TITLE
CLDR-17947 Update sitemap.tsv

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17947

Revises the sitemap to be more semantic (the directory structure got pretty random using Google Sites), and adding some subpages.

Note: PRs that change the sitemap or .md files may break document staging, but there is no check to keep them from being merged. So check results manually before merging.

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
@macchiati i was commenting on the slack channel, but just wondering if this is working for you or if the messages make sense.  Also i had noted how you can run locally.

File: docs/index.md [hunk 1]
```diff
@@ -1,23 +1,5 @@
 ---
-
+title: See Main Page
```
Comment (srl295) [inline docs/index.md:2]: This isn't a site page so not needed. Good change.

**Review by srl295 — APPROVED** [review]
Rubber stamp and sitemap lgtm

File: docs/index.md (hunk 1 above)
Comment (macchiati) [reply docs/index.md:2]: Thanks!!

**Pull request merged**